### DISCUSSION TITLE
Edit WinUI 3 CheckBox Template

### DISCUSSION THREAD

**Discussion by MajidAbuRmelah** [microsoft/microsoft-ui-xaml#7589] [Q&A] [answered]
Hello,

I would like to do my own changes on teh CheckBox template but I am facing an issue editing the WinUI 3 CheckBox template.  I assume that the control and it's styles are in this file.
[https://github.com/microsoft/microsoft-ui-xaml/blob/main/dev/CommonStyles/CheckBox_themeresources.xaml](url)

I get an error at the AnimatedControl in this property `contract7NotPresent:Opacity="0"`.

> Error	XLS0413	The property 'Opacity' was not found in type 'AnimatedIcon'.
Error	XDG0032	The property "Opacity" does not exist in the "http://schemas.microsoft.com/winfx/2006/xaml/presentation" 

How can I solve this issue? I would like to preserve the WinUI 3 styling including the 'check' animation.

**Comment by castorix** [comment]
You can copy and modify the style from _generic.xaml_ :

```
 <Style x:Key="DefaultCheckBoxStyle" TargetType="CheckBox">
 //...
 </Style>
```

You must add :

    xmlns:controls="using:Microsoft.UI.Xaml.Controls"
    xmlns:animatedvisuals="using:Microsoft.UI.Xaml.Controls.AnimatedVisuals"

**Comment by krschau** [comment]
@MajidAbuRmelah Did @castorix's suggestion work for you?

**Reply by MajidAbuRmelah** [reply]
> @MajidAbuRmelah Did @castorix's suggestion work for you?

His suggestion did not work unfortunately.
I solved the issue by replacing `contract7NotPresent:Opacity="0"` with `Opacity="1"`.

Thank you for checking in.

**Comment by MajidAbuRmelah** [comment, marked as answer]
Replace `contract7NotPresent:Opacity="0"` with `Opacity="1"`.
